Decred Wallet/view_controller/ReceiveViewController.swift

```diff
@@ -92,6 +112,30 @@ class ReceiveViewController: UIViewController,UIDocumentInteractionControllerDel
         self.present(alertController, animated: true, completion: nil)
     }
     
+    // MARK: - Private instance methods
+
+    private func toggleView() {
+        if isNewWalletSetup && !isSynced {
+            contentStackView.isHidden = false
+        } else {
+            if isSynced {contentStackView.isHidden = !contentStackView.isHidden}
+        }
+    }
+
+    private func configureProgressLabel() {
```

Comment:
This doesn't really `configure` the progress label. I'm not really quite sure what it does yet but it looks quite similar to the preceding `toggleView` function. See if both functions can be merged or probably rename this function.